pkg/controller/volume/attachdetach/reconciler/reconciler_test.go

```diff
@@ -75,7 +75,7 @@ func Test_Run_Positive_DoNothing(t *testing.T) {
 		fakeKubeClient, informerFactory.Core().V1().Nodes().Lister(), asw)
 	nodeLister := informerFactory.Core().V1().Nodes().Lister()
 	reconciler := NewReconciler(
-		reconcilerLoopPeriod, maxWaitForUnmountDuration, syncLoopPeriod, false, dsw, asw, ad, nsu, nodeLister, fakeRecorder)
+		reconcilerLoopPeriod, maxWaitForUnmountDuration, syncLoopPeriod, false, false, dsw, asw, ad, nsu, nodeLister, fakeRecorder)
```

Comment:
wondering whether you could set a global var instead of adding false in all places. In the future if this is changed, you only need to change one place instead?

Reply:
global variable is not good for unit tests that can run in parallel. Another option is to refactor the `NewReconciler` function to take an options struct with default values. But I think that is out of scope for this change.

Reply:
Thanks; the I'm marking this as resolved for now.